core/pkg/server/server.go

```diff
@@ -105,6 +148,9 @@ func (s *Server) Wait() {
 
 // Close closes the server
 func (s *Server) Close() {
+	<-s.teardownChan
+	close(s.pidwatchChan)
+	close(s.shutdownChan)
```

Comment:
<img width="354" alt="image" src="https://github.com/wandb/wandb/assets/7557205/e8b026ee-ef74-41ba-ac4b-66e78aef332a">

Reply:
yes, this was a merge artifact, removed